ee/clickhouse/queries/experiments/utils.py

```diff
@@ -26,7 +26,9 @@ def requires_flag_warning(filter: Filter, team: Team) -> bool:
     for entity in entities_to_use:
         if entity.type == TREND_FILTER_TYPE_ACTIONS:
             action = entity.get_action()
-            events.update(action.get_step_events())
+            for step_event in action.get_step_events():
```

Comment:
@neilkakkar this one is one of the only changes I'm not sure about. How should it account for "All events" i.e. None here?

Reply:
hmm, indeed, if 'All events' is here this will bork, in this case there ought to be no event filter, i.e. the `entity_query` should be `AND 1=1`.

Since this is already broken, upto you to merge as-is and make an issue for us, or fix it here

Reply:
Hmmmm if its okay I will leave it as a comment and issue so as not to make this already big PR too much

Reply:
yep all good